xontrib/vox.py

```diff
@@ -1,48 +1,41 @@
 """Python virtual environment manager for xonsh."""
 
-import sys
-import textwrap
 import xontrib.voxapi as voxapi
-import xonsh.lazyasd as lazyasd
+from xonsh.built_ins import XSH
+from xonsh.cli_utils import Annotated, Arg, ArgParserAlias
```

Comment:
In #4454 you are using a different import scheme for the `cli_utils` -- it should be consistent across all usage to avoid double-counting in amalgamation

Reply:
Yes I tried to keep 'import ... as' but it raised error sometimes during amalgamation. I will try changing it again.

Reply:
I think those times are when the module level is same as `xonsh.*` . That makes sense since all those definitions will end up in single module. I think we can keep this style in other module levels.